vsts/vsts.yaml

```diff
@@ -494,7 +512,7 @@ jobs:
           E2E_IKEY: $(E2E-IKEY)
 
           # Environment variables for the DevOps pipeline
-          PROXY_SERVER_ADDRESS: 127.0.0.1:3128
+          PROXY_SERVER_ADDRESS: 127.0.0.1:8888
```

Comment:
I think we have the proxy set in a keyvault setting, but I could be wrong. Should we pull it out of there?

Reply:
you'll want to check the proxy server address for both linux (docker image) and windows (squid proxy setup). If they're the same then we can put the value in keyvault. If they're different then we should hardcode it in our yaml directly and add a note explaining why.

Reply:
Linux and windows have different proxy and they have different ports.
For windows, we install SQUID and use the default setup. The proxy port should be 3128.
If indeed we want to change this port, squid/squid.conf need to setup with this port number.